packages/devtools_app/lib/src/config_specific/flutter/framework_initialize/_framework_initialize_stub.dart

```diff
@@ -0,0 +1,8 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
```

Comment:
nit: rename file to _fake_framework_initialize to match other config specific files.

Reply:
I think this naming pattern is better. I'd like to update the other libraries to match this unless there is a good reason for the current convention.
With _fake last all the implementations will have a common prefix which makes it easier to catch typos in the names, etc.